### PR TITLE
Update icons

### PR DESCRIPTION
**Pull request by coderitual**
- Add new web icon - `send`
Screenshot 2021-03-23 at 09 17 11

- Rename mobile icon - `sent` → `arrow`
Screenshot 2021-03-23 at 09 18 03

### CODE REVIEW

File: src/images/icons/send.svg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">
+<title>send </title>
```
Comment (matzimowski) [inline src/images/icons/send.svg:2]: unnecessary whitespace! 👁️

**Review by matzimowski — APPROVED** [review]

File: src/images/icons/send.svg (hunk 1 above)
Comment (coderitual) [reply src/images/icons/send.svg:2]: oh, thanks

**Review by ElaLukaszewicz — APPROVED** [review]
